code/modules/mob/living/simple_animal/hostile/giant_spider/spiderling.dm

```diff
@@ -39,6 +39,7 @@
 	wanted_objects = list(/obj/machinery/atmospherics/unary/vent_pump)
 
 	environment_smash = 0//spiderlings cannot smash tables and windows anymore when getting stomped
+	var/list/spider_types = list(/mob/living/simple_animal/hostile/giant_spider, /mob/living/simple_animal/hostile/giant_spider/nurse, /mob/living/simple_animal/hostile/giant_spider/hunter)
```

Comment:
I'm not sure if BYOND can make lists fully static, but even if it can, you don't want to. It would break `is_type_in_list()`